client/templates/includes/sweepParameterRange.pug

```diff
@@ -7,12 +7,12 @@ div
 
     div.inline.mr-2=this.model.name + ":"
 
-    div.inline(data-hook=this.model.elementID + "-current-value")=this.parameter.expression
+    div.inline(data-hook=this.model.elementID + "-current-value")=this.model.range[0]
```

Comment:
When the current value is a long decimal like `0.07000000000001`, it spills off the screen at certain screen sizes. May want to round the display?